samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -732,89 +677,116 @@ static bool find_valid_bis_in_subgroup_bis(const struct bt_bap_base_subgroup *su
  *    configuration overwrites the subgroup values
  *
  * This function returns `true` if the subgroup does not support the channels in
- * CONFIG_TARGET_BROADCAST_CHANNEL which makes it iterate over the next subgroup, and returns
- * `false` if this subgroup satisfies our CONFIG_TARGET_BROADCAST_CHANNEL.
+ * CONFIG_TARGET_BROADCAST_CHANNEL which makes it iterate over the next subgroup, and
+ * returns `false` if this subgroup satisfies our CONFIG_TARGET_BROADCAST_CHANNEL.
  */
-static bool find_valid_bis_in_subgroup_cb(const struct bt_bap_base_subgroup *subgroup,
-					  void *user_data)
+static bool subgroup_get_valid_bis_indexes_cb(const struct bt_bap_base_subgroup *subgroup,
+					      void *user_data)
 {
+	enum bt_audio_location subgroup_chan_allocation = BT_AUDIO_LOCATION_MONO_AUDIO;
+	enum bt_audio_location bis_chan_allocation = BT_AUDIO_LOCATION_MONO_AUDIO;
```

Comment:
Instead of instantiating these values, you can actually just call `bt_audio_codec_cfg_get_chan_allocation` with `fallback` to `true` which has the same behavior